src/metabase/api/permission_graph.clj

```diff
@@ -55,14 +55,14 @@
 
 (def ^:private TablePerms
   [:or
-   [:enum :all :segmented :none :full :limited :unrestricted :sandboxed :query-builder :no]
+   [:enum :all :segmented :none :full :limited :unrestricted :legacy-no-self-service :sandboxed :query-builder :no]
    [:map
     [:read {:optional true} [:enum :all :none]]
     [:query {:optional true} [:enum :all :none :segmented]]]])
 
 (def ^:private SchemaPerms
   [:or
-   [:keyword {:title "schema name"}]
+   [:enum :all :segmented :none :full :limited :unrestricted :legacy-no-self-service :sandboxed :query-builder :no]
```

Comment:
Is this list def'd out somewhere? I recall some `def`s where we listed out similar sorts of data.

Reply:
Maybe we should def it somewhere else. Either way it doesn't need to happen in this PR though.

Reply:
Ideally we'd have the API graph be the same as the backend graph and share the perm definitions for both. I think we can get to that place soon, but for the moment I think this just needs to be defined separately.